contracts/IDutchAuction.sol

```diff
@@ -0,0 +1,28 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+interface IDutchAuction {
+    error InvalidStartEndTime();
+    error NotStarted();
+    error NotEnded();
+    error InvalidAmountInWei();
+    error NotRefundable();
+    error TransferFailed();
+    error UserAlreadyClaimed();
+
+    struct User {
+        uint216 contribution; // cumulative sum of Wei bids
+        uint32 tokensBidded; // cumulative sum of bidded tokens
+        bool refundClaimed; // has user been refunded yet
+    }
+
+    struct Config {
+        uint256 startAmountInWei;
+        uint256 endAmountInWei;
+        uint256 startTime;
+        uint256 endTime;
+        bool roundUp;
```

Comment:
do we need this to be an option? maybe we can just always round up?

Reply:
yes, good idea